src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -240,6 +240,9 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 	<AndroidSdkToolsVersion Condition="'$(AndroidSdkToolsVersion)' == ''">26.1.1</AndroidSdkToolsVersion>
 	<AndroidNdkVersion Condition="'$(AndroidNdkVersion)' == ''">16.1</AndroidNdkVersion>
 
+	<AndroidDebugKeyAlgorithm Condition=" '$(AndroidDebugKeyAlgorithm)' == '' ">RSA</AndroidDebugKeyAlgorithm>
+	<AndroidDebugKeyValidity Condition=" $(AndroidDebugKeyValidity)'' == '' ">10950</AndroidDebugKeyValidity>
```

Comment:
This is improperly quoted, and results in a [build error](https://jenkins.mono-project.com/job/xamarin-android-pr-builder/2622/console):

```
Unable to parse condition " $(AndroidDebugKeyValidity)'' == '' " : Unexpected token found, "" at character position 28, in condition " $(AndroidDebugKeyValidity)'' == '' "
```